### PR TITLE
Remove cursor: pointer from tree

### PR DESCRIPTION
**Pull request by lintaho**
Associated Issue: #3789

### Summary of Changes

Removes cursor pointer from tree for more cursor consistency

### Test Plan

visual test on clicking tree

### Screenshots/Videos (OPTIONAL)

### CODE REVIEW

**Comment by lintaho** [comment]
@jasonLaster

File: packages/devtools-components/src/tree.css [hunk 1]
```diff
@@ -37,7 +37,7 @@
 }
 
 .tree .tree-node[data-expandable="true"] {
-  cursor: pointer;
+  cursor: default;
```
Comment (nchevobbe) [inline packages/devtools-components/src/tree.css]: what about removing this rule completely ? It should give us the default cursor

**Comment by lintaho** [comment]
hey @nchevobbe - removing it would turn it into cursor: auto, which would show the text cursor instead of the normal arrow when hovering over text

**Comment by nchevobbe** [comment]
oh right, nevermind then :)

**Comment by nchevobbe** [comment]
And it's merged, thanks @lintaho ! (great avatar BTW :) )

**Pull request merged**